.gitlab-ci.yml

```diff
@@ -0,0 +1,82 @@
+stages:
+  - build
+  - test
+  - push
+
+variables:
+  COMPOSE_PROJECT_NAME: "kitsune-${CI_JOB_ID}-${CI_COMMIT_SHORT_SHA}"
```

Comment:
This is interesting! Can you explain a bit more the reason to change the project name?

Reply:
This allows multiple runs of the same docker-compose on the same machine since the test jobs will run in parallel. The job ID being in there keeps them separate and allows them to be easily identified and killed after.

Reply:
See https://docs.docker.com/compose/reference/envvars/#compose_project_name